includes/notifications/views/class.llms.notification.view.upcoming.payment.reminder.php

```diff
@@ -219,7 +220,9 @@ protected function set_merge_data( $code ) {
 
 			case '{{PRODUCT_TYPE}}':
 				$obj = $order->get_product();
-				if ( is_a( $obj, 'WP_Post' ) ) {
+				if ( empty( $obj ) ) {
+					$code = __( '[DELETED PRODUCT]', 'lifterlms' );
```

Comment:
Let's follow the generic "Item" used below

```suggestion
					$code = __( '[DELETED ITEM]', 'lifterlms' );
```